src/main.cc

```diff
@@ -451,6 +452,13 @@ load_cfg(void)
 		}
 	}
 
+	bool flightrec_is_enabled = cfg_getb("flightrec_enabled") == 1;
```

Comment:
Nit: `== 1` is redundant.

Reply:
cfg_getb() can return -1 in case of nil. In this case flight recorder should be inactive (-1 is true when converted to boolean).